frameworks/elastic/universe-kibana/marathon.json.mustache

```diff
@@ -16,9 +16,9 @@
     "FRAMEWORK_USER": "{{service.user}}",
     "XPACK_ENABLED": "{{kibana.xpack_enabled}}",
     "ELASTICSEARCH_URL": "{{kibana.elasticsearch_url}}",
-    {{#service.tls}}
+    {{#service.security.tls.enabled}}
```

Comment:
Note that I have filed https://jira.mesosphere.com/browse/INFINITY-2720 to track the apparent inconsistency here.